pkg/queue/constants.go

```diff
@@ -28,12 +28,18 @@ const (
 	// accepted requests have been processed.
 	RequestQueueDrainPath = "/wait-for-drain"
 
-	// CertDirectory is the name of the directory path where certificates are stored.
-	CertDirectory = "/var/lib/knative/certs"
+	// CertVolumeMountPath is the name of the directory path where certificates are stored.
+	CertVolumeMountPath = "/var/lib/knative/certs"
```

Comment:
minor nit pick: From a user's point of view, it does not matter whether a directory is mounted or part or a original directory of that file directory. Therefore I really would prefer "directory" instead of "mountPath" except when you would actually would do the mount yourself (I doubt that).  I would better rename the other MountPath constants to use Directory instead (like PodInfoVolumeMountPath)

Reply:
Fine either way, as long as all mout points use the same convention.

Note however that we added a config feature to control if podInfo is mounted or not - this is also reflected in the documentation as a result - so we do expose this externally as a mounted volume. 

If we open a new PR for this, I would like to also add a PodInfoAnnotationPath const in shardmain  - this allows Options importing shardmain to use the shardmain const rather than importing also pkg/queue to gain access to the consts there.

Reply:
"directory" works for me (it also seems more in line with the spirit of these variables, as the original comments note these are for "the name of the directory path")